### PR TITLE
add GetDuration to SimpleMotionComponentBus

### PR DESCRIPTION
**Pull request by CyaNinix**

### CODE REVIEW

File: Gems/EMotionFX/Code/Source/Integration/Components/SimpleMotionComponent.cpp [hunk 1]
```diff
@@ -355,6 +355,16 @@ namespace EMotionFX
...
         }
 
+        float SimpleMotionComponent::GetDuration() const
+        {
+            float result = 0.0f;
+            if (m_motionInstance)
+            {
+                result = m_motionInstance->GetDuration();
+            }
+            return result;
```
Comment (hultonha) [inline Gems/EMotionFX/Code/Source/Integration/Components/SimpleMotionComponent.cpp:365]: suggestion: We could update this to (same below)

```
return m_motionInstance ? m_motionInstance->GetDuration() : 0.0f;
```

**Review by hultonha — COMMENTED** [review]
Looks good to me! 🙂 Just one small suggestion, Thanks! Adding @amzn-rhhong and @amzn-jillich to take a look too 👍

**Comment by AMZN-alexpete** [comment]
Thanks for submitting this @CyaNinix , can you please also add the new function to the BehaviorContext declaration in SimpleMotionComponent::Reflect(), e.g.
```
   ->Event("GetDuration", &SimpleMotionComponentRequestBus::Events::GetDuration)
```

**Review by moraaar — APPROVED** [review]
LGTM. Thanks for the contribution!

Only one suggestion, EMotionFX gem has very good unit test coverage, I'd recommend adding a simple unit test to `o3de\Gems\EMotionFX\Code\Tests\SimpleMotionComponentBusTests.cpp` to check that the new `GetDuration` function is working as expected.

**Review by AMZN-alexpete — APPROVED** [review]

**Review by amzn-jillich — APPROVED** [review]

**Comment by AMZN-alexpete** [comment]
For some reason on Jenkins, the [asset processor keeps timing out on Linux ](https://jenkins.build.o3de.org/blue/rest/organizations/jenkins/pipelines/O3DE/branches/PR-10038/runs/3/nodes/80/log/?start=0).  I'm trying to clean the whole build, but if that doesn't work you may need to merge latest from dev to fix whatever gremlins are lurking in the particular version of dev is in this branch.

**Pull request merged**